src/tests/CullingTest.cpp

```diff
@@ -88,17 +105,17 @@ TEST(CullingTest, computeAbsoluteAABB) {
       {{3.0f, -1.0f, 3.0f}, {5.0f, 1.0f, 5.0f}}};
 
   // compare against the ground truth
-  EXPECT_EQ(aabbs.size(), aabbsGroundTruth.size());
-  const float epsilon = 1e-6;
+  CORRADE_COMPARE(aabbs.size(), aabbsGroundTruth.size());
+  const float eps = 1e-6;
   for (unsigned int iBox = 0; iBox < aabbsGroundTruth.size(); ++iBox) {
```

Comment:
You may want to use [`CORRADE_ITERATION()`](https://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#TestSuite-Tester-iteration-annotations) here for better failure diagnostic:

```cpp
  for (unsigned int iBox = 0; iBox < aabbsGroundTruth.size(); ++iBox) {
    CORRADE_ITERATION(iBox);
    …
```